resources/make/common.mk

```diff
@@ -106,16 +123,6 @@ push-all:
 	git push upstream --tags
 
 install: compile
-	@echo "Installing calrissian ..."
+	@echo "Installing lumberjack ..."
```

Comment:
Ha! Thanks for the catch -- I'll clean up the bad copy-paste.


On Thu, May 21, 2015 at 7:20 PM, Correl Roush <notifications@github.com>
wrote:

> In resources/make/common.mk
> <https://github.com/correl/calrissian/pull/3#discussion_r30861293>:
>
> > @@ -106,16 +123,6 @@ push-all:
> >  	git push upstream --tags
> >
> >  install: compile
> > -	@echo "Installing calrissian ..."
> > +	@echo "Installing lumberjack ..."
>
> Quite a change, probably not intentional :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/correl/calrissian/pull/3/files#r30861293>.
>